doc/apiary/v2/fiware-ngsiv2-rc-2016_05_31.apib

```diff
@@ -62,6 +66,11 @@ NGSI version 2 uses camel case (camelCase) syntax for naming properties and rela
 by the API. When  referring to URIs, as part of HATEOAS patterns, and to mark them appropriately,
 the suffix `_url` is added.
 
+## Reference Implementations
+
+* NGISv2 servers
+  * [Orion Context Broker](http://catalogue.fiware.org/enablers/publishsubscribe-context-broker-orion-context-broker) - [Implementation Notes](https://fiware-orion.readthedocs.io/en/develop/user/ngsiv2_implementation_notes/index.html)
```

Comment:
URL makes this line very long and I'm a bit unsure how breaking this into two lines could affect to the bullet list indentation.

Thus, this time, I prefer to leave it long :)